build.gradle

```diff
@@ -28,6 +28,10 @@ allprojects {
     configureDokka(it)
 }
 
+subprojects {
+    task allDeps(type: DependencyReportTask) {}
+}
```

Comment:
I decided to add this task here to make my life simpler with dependency scanning. This only executes if I explicitly call `gradlew allDeps`